app/src/androidTest/java/com/example/admin/personallibrarycatalogue/data/TestDb.java

```diff
@@ -0,0 +1,102 @@
+package com.example.admin.personallibrarycatalogue.data;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.test.AndroidTestCase;
+import android.util.Log;
+
+import com.example.admin.personallibrarycatalogue.R;
+import com.example.admin.personallibrarycatalogue.data.DatabaseContract.BooksTable;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+
+/**
+ * Created by Mikhail Valuyskiy on 26.05.2015.
+ */
+public class TestDb extends AndroidTestCase {
+
+    public static final String LOG_TAG = TestDb.class.getSimpleName();
+
+    private static final String BOOK_TITLE = "Thinking in Java";
+    private static final String BOOK_AUTHOR = "Bruce Eckel";
+    private static final String BOOK_DESCRIPTION = "Thinking in Java is a book about the Java programming language, written by Bruce Eckel and first published in 1998";
+
+    private LibraryDatabaseHelper helper_;
+    private SQLiteDatabase database_;
+
+    @Override
+    protected void setUp() throws Exception {
+        super.setUp();
+        helper_ = new LibraryDatabaseHelper(mContext);
+        database_ = helper_.getWritableDatabase();
+    }
+
+    @Override
+    protected void tearDown() throws Exception {
+        super.tearDown();
+        mContext.deleteDatabase(LibraryDatabaseHelper.DATABASE_NAME);
+        helper_.close();
```

Comment:
1. Why do you close DB after deletion? Guess close and deletion should be switched in order.
2. It's better to use ```helper.deleteDB``` instead of ```mContext.deleteDatabase```.
3. ```helper.deleteDB``` may already contain ```DB.close```.